spec/integ/crypto/crypto.spec.ts

```diff
@@ -670,18 +670,21 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
     });
 
     it("prepareToEncrypt", async () => {
-        expectAliceKeyQuery({ device_keys: { "@alice:localhost": {} }, failures: {} });
+        const homeserverUrl = "https://alice-server.com";
```

Comment:
For the future, maybe: consider `aliceClient.getHomeserverUrl()` rather than hardcoding the server url here.